bots/pr/src/main/java/org/openjdk/skara/bots/pr/ApprovalCommand.java

```diff
@@ -75,91 +72,65 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
         String option = argMatcher.group(3);
         String message = argMatcher.group(4);
 
-        var issueOpt = getIssue(issueId, pr, allComments, reply);
-        if (issueOpt.isEmpty()) {
+        var issues = getIssues(issueId, pr, allComments, reply);
+        if (issues.isEmpty()) {
             return;
         }
-        var issue = issueOpt.get();
-
-        if (issue.project().isPresent() && !issue.project().get().equalsIgnoreCase(issueProject.name())) {
-            reply.println("Approval can only be requested for issues in the " + issueProject.name() + " project.");
-            return;
-        }
-
-        var issueTrackerIssueOpt = issueProject.issue(issue.shortId());
-        if (issueTrackerIssueOpt.isEmpty()) {
-            reply.println("Can not find " + issue.id() + " in the " + issueProject.name() + " project.");
-            return;
-        }
-        var issueTrackerIssue = issueTrackerIssueOpt.get();
-        var requestLabel = approval.requestedLabel(targetRef);
-        var approvedLabel = approval.approvedLabel(targetRef);
-        var rejectedLabel = approval.rejectedLabel(targetRef);
-        var prefix = "[" + requestLabel + "]";
-        var comments = issueTrackerIssue.comments();
-        var existingComment = comments.stream()
-                .filter(comment -> comment.author().equals(issueProject.issueTracker().currentUser()))
-                .filter(comment -> comment.body().startsWith(prefix))
-                .findFirst();
+        reply.println();
+        for (var issue : issues) {
+            reply.print(issue.id() + ": ");
+            if (issue.project().isPresent() && !issue.project().get().equalsIgnoreCase(issueProject.name())) {
+                reply.println("Approval can only be requested for issues in the " + issueProject.name() + " project.");
+                continue;
+            }
 
-        var labels = issueTrackerIssue.labelNames();
-        if (option.equals("cancel")) {
-            if (labels.contains(approvedLabel) || labels.contains(rejectedLabel)) {
-                reply.println("The request has already been handled by a maintainer and can no longer be canceled.");
-            } else {
-                issueTrackerIssue.removeLabel(requestLabel);
-                existingComment.ifPresent(issueTrackerIssue::removeComment);
-                reply.println("The approval request has been cancelled successfully.");
+            var issueTrackerIssueOpt = issueProject.issue(issue.shortId());
+            if (issueTrackerIssueOpt.isEmpty()) {
+                reply.println("Can not find " + issue.id() + " in the " + issueProject.name() + " project.");
```

Comment:
The ID is already printed, so this error message doesn't need to include it. Maybe something like

```suggestion
                reply.println("Can not be found in the " + issueProject.name() + " project.");
```